### ISSUE TITLE
Use redirector to eliminate need for internet connection when installing

### ISSUE THREAD

**Issue by ryecoaaron** [openmediavault/openmediavault#287]
I think the OMV iso should use the redirector for the Debian repo sources eliminating the need for an internet connection during installation.

```
deb http://httpredir.debian.org/debian/ stretch main contrib non-free
deb http://security.debian.org/debian-security stretch/updates main contrib non-free
deb http://httpredir.debian.org/debian/ stretch-updates main contrib non-free
```

**Comment by mi-hol** [comment]
@votdev any comments?

**Comment by mi-hol** [comment]
@votdev over the holidays we've seen many strange errors resulting in corrupt or only partially working installations
[Forum post](https://forum.openmediavault.org/index.php?thread/29876-rpi-and-other-sbc-armbian-images/&postID=259775#post259775)
Would the suggested feature have prevented these issues from happening?

**Comment by votdev** [comment]
> @votdev over the holidays we've seen many strange errors resulting in corrupt or only partially working installations
> 
> [Forum post](https://forum.openmediavault.org/index.php?thread/29876-rpi-and-other-sbc-armbian-images/&postID=259775#post259775)
> 
> Would the suggested feature have prevented these issues from happening?

No, we are talking about the ISO installation for AMD64 here, the problem you are referring to are related to ARM installations based on images.

**Comment by mi-hol** [comment]
Likely many new Pi5 users will be trying to install OMV 6 & 7 during x-mas break period for IT pros.
I'd envisage many instances of installation issues due to unavailable or otherwise broken mirrors.
Any chance to mitigate this via changes in installation procedure?

**Comment by ryecoaaron** [comment]
RPi5 can’t run OMV6 and OMV7 isn’t released but the RPi images already have mirrors setup. So this isnt and RPi issue.
